loxinlp/nlp.go

```diff
@@ -16,805 +16,912 @@
 package loxinlp
 
 import (
-    "fmt"
-    cmn "loxilb/common"
-    tk "loxilb/loxilib"
-    "net"
-    "regexp"
-    "strconv"
-    "strings"
-    "syscall"
-    "time"
-    "os"
-    "os/exec"
-    "errors"
-    nlp "github.com/vishvananda/netlink"
-    "golang.org/x/sys/unix"
+	"encoding/json"
```

Comment:
I think indentation has changed and showing up unnecessary changes. I suggest fixing VsCode's settings from tab to space and update the pull request.

Reply:
Hi, for the collaboration this open source, I think it will follow official formatting guide in the Golang. Please refer to the following link(https://go.dev/doc/effective_go#formatting). Thanks.

Reply:
Thanks @inhogog2 for the comment. For this current pull request,  kindly change to original indentation of the file. Later , lets review and fix the indentation of the whole project.